docs/codeReview.md

```diff
@@ -13,9 +10,10 @@ A PR advances from self-review to reviewer-review and finally maintainer review
 
 ## Requesting Code Reviews
 
-- Self-assess which domains are involved in the changes made by your PR, find a reviewer from among the folks who fill the [Code Reviewer (L2–3)](https://www.notion.so/Code-Reviewer-L2-3-a47bb0759a0b41b5b0469ff14a8cdaae) role, paying specific attention to who GitHub suggests, who has indicated expertise within a domain (e.g. backend, front-end, etc.), and who has capacity to review within 24 hours.
+- Self-assess which domains are involved in the changes made by your PR, find a reviewer from among the folks who are in the reviewers or maintainer [groups](/.github/reviewers.yml), paying specific attention to who GitHub suggests, who has indicated expertise within a domain (e.g. backend, front-end, etc.), and who has capacity to review within 24 hours.
+  - If you feel that maintainer review isn't necessary, add the label: "One Review Required". Use the label for PRs of any size and complexity if you are confident enough, and it feels safe. One exception: we should be extremely careful with DB migrations as they might break the application in the way that is hard to restore.
```

Comment:
So if I understand correctly, basically, we decide if PR is complex or not and request either reviewer or maintainer. Do we need to keep `One Review Required` label in this case if anyway it always be a single review?

Reply:
The intention of "One Review Required" was to bypass maintainer review. I think that's still worth marking as it makes it easier to check after some time if it worked out. I'm open to get new name suggestions.